### ISSUE TITLE
regard cross validtion

### ISSUE THREAD

**Issue by Phd-Student2018** [flairNLP/flair#2290]
Hi
Thanks for sharing this great work
I work on NER, I use  (bert+ crf+ LSTM) but I want to enhance the result using cross validation , how can I modify the code to do that?
Thnaks in advance
this code I used:

tagger: SequenceTagger = SequenceTagger(
        hidden_size=256,
        embeddings=embeddings,
        tag_dictionary=tag_dictionary,
        tag_type=tag_type,
        use_crf=[True],
        #use_rnn=False,
        rnn_type="LSTM",
        rnn_layers=2

trainer = ModelTrainer(tagger, corpus, optimizer=torch.optim.Adam)
trainer.train('resources/taggers/example-pos',
              learning_rate=5e-5,
              embeddings_storage_mode = 'gpu',
              mini_batch_chunk_size=2, # set this if you get OOM errors
              max_epochs=25, # very few epochs of fine-tuning
              #shuffle=False,
              train_with_dev=False,
              checkpoint=True)

plotter = Plotter()
